scripts/west_commands/runners/linkserver.py

```diff
@@ -130,13 +131,15 @@ def do_run(self, command, **kwargs):
 
             linkserver_cmd = ([self.linkserver] +
                               ["gdbserver"]    +
-                              ["--probe", "#"+str(self.probe) ] +
                               ["--gdb-port", str(self.gdb_port )] +
                               ["--semihost-port", str(self.semihost_port) ] +
+                              ([f"--probe={self.dev_id}"] if self.dev_id else []) +
                               _cmd_core +
                               self.override_cli +
                               [self.device])
 
+
+
```

Comment:
Stray newlines?

Reply:
Thanks for noticing- they've been removed